cypress/e2e/tests/migration/applicationinventory/applications/crud.test.ts

```diff
@@ -49,4 +64,22 @@ describe(["@tier1"], "Application crud operations", () => {
         // Assert that newly created application is deleted
         notExists(application.name);
     });
+
+    it("Update application profile details", function () {
+        const stakeholder = new Stakeholders(data.getEmail(), data.getFullName());
+        stakeholder.create();
+        cy.wait(2000);
+        // const application = new Assessment(getRandomApplicationData());
+        const application = new Analysis(
```

Comment:
No need to use json file data here. 
Please create appdata similar to 

Create appData like here and provide values in the test https://github.com/konveyor/tackle-ui-tests/blob/main/cypress/e2e/tests/migration/applicationinventory/applications/interlinked.test.ts#L70C13-L70C20 

with random values  and validate